### PR TITLE
added port and path

### PR DESCRIPTION
**Pull request by CodeSEAS**
The first springboot page launched

### CODE REVIEW

**Review by stargrep — COMMENTED** [review]
can you please make sure not include .gradle/ and .DS_Store files in the commit?
For .gradle files, we can just remove the whole folder.
For .DS_Store can you add to git ignore?

File: .gradle/buildOutputCleanup/cache.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#Thu Feb 04 23:52:32 EST 2021
+gradle.version=6.5
```
Comment (stargrep) [inline .gradle/buildOutputCleanup/cache.properties:2]: No need

File: src/main/java/com/lory/gateway/hello.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.lory.gateway;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class hello {
```
Comment (stargrep) [inline src/main/java/com/lory/gateway/hello.java:8]: class name should be Capitalized

File: src/main/java/com/lory/gateway/hello.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    @RequestMapping(value="/hello", method = RequestMethod.GET) //定义路由
```
Comment (stargrep) [inline src/main/java/com/lory/gateway/hello.java:9]: Please change comments to be English

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -1 +1,2 @@
-
+server.port=8080
+server.servlet.context-path=/lory
```
Comment (stargrep) [inline src/main/resources/application.properties:2]: Don't think we need the extension? or Maybe `api-gateway` is better?

**Review by stargrep — CHANGES_REQUESTED** [review]

**Comment by CodeSEAS** [comment]
The changes have been made according to the recommendations. Thanks:)

**Review by stargrep — APPROVED** [review]

**Comment by stargrep** [comment]
accept to unblock.

**Pull request merged**